packages/clay-css/src/scss/components/_empty-state.scss

```diff
@@ -63,3 +63,242 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
```

Comment:
@pat270 Could you take a look at this?

Reply:
@ilzamcmed we can inline the the styles that don't provide animation in the svg and only include the animation styles in Clay CSS.

Reply:
Thanks @pat270.
Done!